pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -111,14 +117,40 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 	templateData.ExternalStaticDNS = config.BootstrapExternalStaticDNS
 	templateData.ExternalMACAddress = config.ExternalMACAddress
 
-	_, externalURLv6 := externalURLs(config.APIVIPs)
+	// If the user has manually set disableVirtualMediaTLS to False in the Provisioning CR,
+	// then enable TLS in ironic
+	templateData.DisableIronicVirtualMediaTLS = true
+	protocol := "http"
+
+	provisioningCR := struct {
+		spec struct {
+			disableVirtualMediaTLS *bool `yaml:"disableVirtualMediaTLS"`
+		} `yaml:"spec"`
+	}{}
+	provisioningCRFile, err := os.Open("99_baremetal-provisioning-config.yaml") // which directory will this be in?
+	if err != nil {
+		logrus.Warnf("Error in opening Provisioning CR YAML file: %s.", err)
+	}
+	defer provisioningCRFile.Close()
+	if provisioningCRFile != nil {
```

Comment:
This is not needed, we should not proceed if something unexpected happened when opening the file

Reply:
Fixed